### PR TITLE
Added imagetools plugin to TinyMCE

### PR DESCRIPTION
**Pull request by riyadhalnur**
ADDED:
- Added support for the 'imagetools' plugin for TinyMCE.
- The plugin enables users to edit inline image in the tinyMCE editor

screen shot 2015-10-25 at 3 03 11 pm

Sorry about the big diff but all these files are actually TinyMCE files. There was no change made to the core. I just replaced the TinyMCE lib with the latest version.

Essentially, any inline image in the text editor can be edited using this plugin. The image can either be copy/pasted if the paste plugin is enabled or an uploaded image added using the URL if the image plugin is enabled.

### CODE REVIEW

**Comment by celiao** [comment]
Has the documentation also been updated?

**Comment by riyadhalnur** [comment]
just one line to update in the documentation. if a more comprehensive breakdown is needed that can be done too

**Comment by morenoh149** [comment]
@riyadhalnur could you summarize the changes? 330 files is a lot of diffs to review. How does it work? Does the image have to exist already online or does it persist it using a keystone image type or s3?

**Comment by riyadhalnur** [comment]
@morenoh149 Sorry about the big diff but all these files are actually TinyMCE files. There was no change made to the core. I just replaced the TinyMCE lib with the latest version.

Essentially, any inline image in the text editor can be edited using this plugin. The image can either be copy/pasted if the `paste` plugin is enabled or an uploaded image added using the URL if the `image` plugin is enabled.

**Comment by mxstbr** [comment]
Seems like a useful addition – can you point me to the relevant change that is not one of the TinyMCE update @riyadhalnur so I can review and merge this?

**Comment by riyadhalnur** [comment]
@mxstbr Didn't get your question about pointing to the relevant change

**Comment by mxstbr** [comment]
Well, I don't want to dig through 330 changed files to find the one change you personally made – can you point me to that file/those files?

**Comment by mxstbr** [comment]
Oh damn closed on accident and now I can't reopen because

> "The repository that submitted this pull request was closed"

Do you know what's up with that?

**Comment by riyadhalnur** [comment]
No problem. I'll open the same one again.
